.gitignore

```diff
@@ -14,3 +14,5 @@
 /Licenses.toml
 /licenses
 *.run
+*.DS_Store
```

Comment:
These should really be part of your global gitignore configuration. I don't think we should be adding OS and IDE specific things here.

See `core.excludesfile` in https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

Reply:
I see where you are coming from on this but in general I don't agree that omitting these files from the gitignore is worth it. While these files are pretty dumb and it would be nice if there was simpler way to never have them show up, the fact is that some day someone will accidentally commit one of these files to this repository. Since everything here is thoroughly code-reviewed I don't think the file will actually end up being merged to the upstream branch. Having them in gitignore will save a little bit of time and avoid the need for a couple of code review comments and some back-and-forths. The trade-off is ~20 bytes of data in the gitignore to ensure some junk files that no one ever wants to see in this repository will ever end up in it.

Anyway, that's simply my pragmatic view of why I usually put these files in my gitignore. It's just my opinion though, so I've already amended the commit to oblige what is probably already established as consensus in this project. :-)

Reply:
Thanks @stefansundin - yeah, I agree there is no "one right way" here and the way you had it certainly works. Thanks a lot for your work on this PR and updating this!